### ISSUE TITLE
Multinucleated giant cells in skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10002]
**Preferred term label:**
Multinucleated giant cells in skin

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mycetoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000057
- label: Dermal multinucleated giant cells
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: PMID:25284855|PMID:35478968
- def: Presence of giant cells in the dermis. Giant cells are formed by fusion of various cells such as macrophage, epithelioid cells, monocytes, etc., These are multi-nucleated, large in size, and generally present at the site of chronic inflammation and other granulomatous conditions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10002
- synonyms: Multinucleated giant cells in skin(NONE)
